compiler/noirc_frontend/src/hir/comptime/value.rs

```diff
@@ -230,8 +245,9 @@ impl Value {
                     }
                 };
             }
-            Value::Expr(expr) => expr,
-            Value::Pointer(..)
+            Value::Expr(ExprValue::Expression(expr)) => expr,
+            Value::Expr(_)
```

Comment:
We can convert statements into expressions by wrapping them in a block

Reply:
Done!